packages/gatsby-plugin-sharp/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -23,6 +23,19 @@ Object {
 }
 `;
 
+exports[`gatsby-plugin-sharp duotone fixed 2`] = `
```

Comment:
Not sure how this got added but could it be removed? It doesn't seem related to me 🤔